### PR TITLE
unit test for Link, Typography and Spacer

### PR DESCRIPTION
**Pull request by ShawTim**

### CODE REVIEW

**Review by danieledamiani — APPROVED** [review]

File: packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ describe('input', () => {
 
     expect(input).not.toBeNull();
     expect(input).not.toBeNull();
-    expect(input.getAttribute('disabled')).not.toBeNull();
+    expect(input).toHaveAttribute('disabled');
```
Comment (iamvukasin) [inline packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx:66]: I think we can use `expect(input).toBeDisabled()`

File: packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ describe('input', () => {
 
     expect(input).not.toBeNull();
     expect(input).not.toBeNull();
```
Comment (iamvukasin) [inline packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx:52]: Why do we have two `expect(input).not.toBeNull()`?

File: packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx [hunk 3]
```diff
@@ -63,7 +63,7 @@ describe('input', () => {
...
```
Comment (iamvukasin) [inline packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx:65]: Same here

File: packages/tempus-client_v3/src/components/shared/NumberInput/NumberInput.spec.tsx [hunk 4]
```diff
@@ -61,7 +61,7 @@ describe('NumberInput', () => {
     expect(input).not.toBeNull();
     expect(button).not.toBeNull();
     expect(label).not.toBeNull();
-    expect(input.getAttribute('disabled')).not.toBeNull();
+    expect(input).toHaveAttribute('disabled');
```
Comment (iamvukasin) [inline packages/tempus-client_v3/src/components/shared/NumberInput/NumberInput.spec.tsx:64]: Same here

File: packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx (hunk 1 above)
Comment (ShawTim) [reply packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx]: fixed

File: packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx (hunk 2 above)
Comment (ShawTim) [reply packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx]: fixed

Comment (ShawTim) [reply packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx]: fixed

File: packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx (hunk 3 above)
Comment (ShawTim) [reply packages/tempus-client_v3/src/components/shared/BaseInput/BaseInput.spec.tsx]: fixed

File: packages/tempus-client_v3/src/components/shared/NumberInput/NumberInput.spec.tsx (hunk 4 above)
Comment (ShawTim) [reply packages/tempus-client_v3/src/components/shared/NumberInput/NumberInput.spec.tsx]: fixed

**Pull request merged**